src/renderer/views/multiplayer/custom.vue

```diff
@@ -226,6 +240,10 @@ async function attemptJoinBattle(battle: SpadsBattle) {
     if (battle.battleOptions.passworded) {
         passwordPromptOpen.value = true;
     } else {
+        loading.value = true;
+        // if user is in a other lobby, leave it
+        api.session.onlineBattle.value && (await api.comms.request("c.lobby.leave"));
```

Comment:
I'm not a huge fan of inline conditional expressions like this, would prefer:
```typescript
if (api.session.onlineBattle.value) {
    await api.comms.request("c.lobby.leave");
}
```

Reply:
changed